docker/Dockerfile

```diff
@@ -1,3 +1,7 @@
+# For Running NVIDIA FLARE in a Docker container, see
+# https://nvflare.readthedocs.io/en/main/quickstart.html#containerized-deployment-with-docker
+# This Dockerfile is primarily for building Docker images to publish for dashboard.
+
 FROM python:3.8
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install zip
```

Comment:
what is the purpose of this docker, to build a dashboard image?
Should we use FROM python:3.12?

Reply:
@IsaacYangSLA said that this was for building the docker images to push to dockerhub for the dashboard, I am not sure if we should update the python version now or at a future time

Reply:
ok we can also do this in a different PR. @IsaacYangSLA ^